include/ebpf_nethooks.h

```diff
@@ -265,6 +265,20 @@ typedef struct _bpf_sock_ops
 typedef int
 sock_ops_hook_t(bpf_sock_ops_t* context);
 
+typedef enum _process_operation
+{
+    PROCESS_OPERATION_CREATE, ///< Process creation.
+    PROCESS_OPERATION_DELETE, ///< Process deletion.
+} process_operation_t;
+
+typedef struct _process_md
+{
+    uint8_t* process_id_start;     ///< Pointer to start of App ID.
+    uint8_t* process_id_end;       ///< Pointer to end of App ID.
```

Comment:
Seems odd to have fields named "process" id if they're "App" id.

Reply:
Good point. Switching this over to passing the full command line + arguments (it's what the customer is asking for).